OpenRA.Game/Map/MapPreview.cs

```diff
@@ -284,9 +298,10 @@ public MapPreview(Map map, ModData modData)
 		{
 			this.modData = modData;
 			cache = modData.MapCache;
+			this.pPackage = null;
```

Comment:
this is null by default, no need to set it

Reply:
also rename to just `package`, there's no need for a special name

Reply:
ok